intermediate_source/rpc_tutorial.rst

```diff
@@ -441,11 +441,15 @@ takes a GPU tensor, you need to move it to the proper device explicitly.
         def __init__(self, ntoken, ninp, dropout):
             super(EmbeddingTable, self).__init__()
             self.drop = nn.Dropout(dropout)
-            self.encoder = nn.Embedding(ntoken, ninp).cuda()
+            self.encoder = nn.Embedding(ntoken, ninp)
+            if torch.cuda.is_available():
+                self.encoder = self.encoder.cuda()
             self.encoder.weight.data.uniform_(-0.1, 0.1)
 
         def forward(self, input):
-            return self.drop(self.encoder(input.cuda()).cpu()
+            if torch.cuda.is_available():
```

Comment:
If we add this it will allow GPU-less users to run the example and it will work as before for GPU owners